sw/device/silicon_creator/mask_rom/sig_verify_keys.c

```diff
@@ -8,100 +8,60 @@
  * Public keys for signature verification.
  */
 const sigverify_rsa_key_t kSigVerifyRsaKeys[kSigVerifyNumRsaKeys] = {
+    // sw/device/silicon_creator/keys/dev_key_0.public.der
```

Comment:
This currently manual some manual steps. Modulus and exponent from openssl commands are copied over (for now I pipe modulus output to `sed 's/Modulus=//' | sed 's/.\{8\}/\L0x&, /g' | tac` to convert to little-endian), and compute `n0_inv` using `hex(pow(n[0], -1, 2**32))` in python. IMHO, it would be great if we could do all of these in `opentitantool`.

Reply:
Please add a comment for now. You don't have to provide a detailed shell command pipeline, just enough detail that a person of reasonable skill can figure it out :)

Reply:
Done.